source/clear-linux/get-started/virtual-machine-install/hyper-v.rst

```diff
@@ -12,26 +12,19 @@ Please ensure you have enabled `Intel® Virtualization Technology
 <https://software.intel.com/en-us/articles/intel-virtualization-technology-for-directed-io-vt-d-enhancing-intel-platforms-for-efficient-virtualization-of-io-devices>`_
 (Intel® VT-d) in your BIOS/UEFI firmware configuration.
 
-Enable Hyper-V
-==============
+Install Hyper-V
+================
 
-Please refer to the `Microsoft documentation`_ to enable and configure
+Please refer to the `Microsoft documentation`_ to install and configure
 *Hyper-V* on your machine.
 
-Create a virtual network
-========================
-
-Once *Hyper-V* has been enabled on your Windows system you will need to 
-create a virtual network in the **Hyper-V Manager**.  Refer to the
-`Create a Virtual Network documentation`_ to create and configure
-a virtual network.
-
 Create a virtual machine
 ========================
 
 #. Download and uncompress the latest hyperv disk image
-   :file:`clear-XXXXX-hyperv.img.gz`, where XXXXX is the latest 
-   available version of |CLOSIA| from our `downloads`_ section.
+   :file:`clear-XXXX-hyperv.img.gz`, where XXXX is the latest 
```

Comment:
the current length of a version string is 5 characters, which is why I changed this.